polynote-kernel/src/main/scala/polynote/kernel/util/RuntimeSymbolTable.scala

```diff
@@ -29,7 +30,7 @@ final class RuntimeSymbolTable(
 
   private val newSymbols: Topic[IO, RuntimeValue] =
     Topic[IO, RuntimeValue]{
-      val kernel = RuntimeValue("kernel", polynote.runtime.Runtime, global.typeOf[polynote.runtime.Runtime.type], None, -1)
+      val kernel = RuntimeValue("kernel", polynote.runtime.Runtime, None, -1)
```

Comment:
hmm, won't this cause an `inferType` for `kernel`? Maybe keep the `global.typeOf` and just manually create a `StringRepr` to pass in (even empty string repr would be fine for the kernel object I think)

Reply:
Done! For some reason IntelliJ gives me angry red underlines when I do that but it compiles fine... 🤷‍♂️